incubator/solr/README.md

```diff
@@ -0,0 +1,224 @@
+# Apache SOLR Cloud Helm Chart
+
+This helm chart provides an implementation of Apache Solr [StatefulSet](http://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/).
+
+<!-- TOC -->
+
+- [Apache SOLR Cloud Helm Chart](#apache-solr-cloud-helm-chart)
+    - [Prerequisites](#prerequisites)
+    - [Chart Components](#chart-components)
+    - [Installing the Chart](#installing-the-chart)
+        - [Installed Components](#installed-components)
+    - [Configuration](#configuration)
+    - [Default Values](#default-values)
+    - [Deep Dive](#deep-dive)
+        - [Images](#images)
+            - [Zookeeper Image](#zookeeper-image)
+            - [Solr Image](#solr-image)
+        - [Scaling](#scaling)
+            - [Solr](#solr)
+            - [Zookeeper](#zookeeper)
+        - [Access API and Client Searching](#access-api-and-client-searching)
+        - [Create a Collection](#create-a-collection)
+        - [Add Data to Collection](#add-data-to-collection)
+        - [Search Data in Collection](#search-data-in-collection)
+
+<!-- /TOC -->
+
+## Prerequisites
+
+- Kubernetes 1.8+
+- PersistentVolume support
+- A familiarity with [Apache Solr 7](http://lucene.apache.org/solr/)
+- Helm Chart Zookeeper 1.2.0 see [requirements.yaml](requirements.yaml)
```

Comment:
I don't think this should be listed as a prerequisite, it's installed automatically as a dependent chart right?